test/parallel/test-tls-honorcipherorder.js

```diff
@@ -46,54 +45,50 @@ function test(honorCipherOrder, clientCipher, expectedCipher, cb) {
       coptions.ciphers = clientCipher;
     }
     const port = this.address().port;
-    const client = tls.connect(port, localhost, coptions, function() {
+    const savedDefaults = tls.DEFAULT_CIPHERS;
+    tls.DEFAULT_CIPHERS = defaultCiphers || savedDefaults;
+    const client = tls.connect(port, localhost, coptions, mustCall(function() {
       const cipher = client.getCipher();
       client.end();
       server.close();
-      assert.strictEqual(cipher.name, expectedCipher);
-      if (cb) cb();
-    });
-  });
+      const msg = fmt(
```

Comment:
Can you use util.format here?  Saves the casual reader from having to look up what fmt() is.